journal/src/main/scala/com/evolutiongaming/kafka/journal/Journals.scala

```diff
@@ -92,22 +95,25 @@ object Journals {
     consumer: Resource[F, Consumer[F]],
     eventualJournal: EventualJournal[F],
     headCache: HeadCache[F],
+    consumerPoolSize: Int,
+    consumerPoolMetrics: Option[ConsumerPoolMetrics[F]],
     log: Log[F],
     conversionMetrics: Option[ConversionMetrics[F]]
-  )(implicit F: MonadCancel[F, Throwable]): Journals[F] = {
+  )(implicit F: GenConcurrent[F, Throwable]): Resource[F, Journals[F]] = {
```

Comment:
As this method now returns `Resource[F, Journals[F]]` instead of `Journals[F]` it makes sense to rename it from `apply` to `make` to avoid confusing the library users.

Reply:
I see you changed it to `of` now. I am fine to keep it `of`, though I have a (false?) memory that the usual style is the following in case you have some class such as `MyClass`:
1. Use `apply` if method returns `MyClass`.
2. Use `of` if method returns `F[MyClass]`.
3. Use `make` if method returns `Resource[F, MyClass]`.

I am not sure about it, so it is okay to keep `of`.

Reply:
I don't have a preference for `of`/`make`. Let's change it to `make` if this is the style we follow. I called it `of` because we already have an `of` method in `Journals` that returns a `Resource` and I thought it would be weird that one method is called `of` and another is called `make`